python/requirements.txt

```diff
@@ -15,7 +15,7 @@ cfn-lint==0.39.0
 chardet==3.0.4
 click==7.1.2
 coverage==5.3
-cryptography==3.2.1
+cryptography==3.3.2
```

Comment:
this package might actually not be used at all

Reply:
You're right -- this is just Python's package manager being shitty. I think we 'pinned' package versions by doing `pip freeze > requirements.txt`. I assume it's a second-order dependency, for the purposes of this we can remove it and it should work just fine.